v4/source/introduction.yaml

```diff
@@ -214,23 +214,192 @@ tags:
+               'total_msg_count': '<total_msg_count>',
+               'extra_update_at': '<timestamp>',
+               'creator_id': '<basic_user_id>'
+             },
+           'broadcast': {
+             'omit_users': null,
+             'user_id': '',
+             'channel_id': '<channel_id>',
+             'team_id': '<team_id>'
+           },
+         }
+         ```
       - channel_viewed
+        ```
+        {
+          'event': 'channel_viewed',
+          'seq': '<seq_id>'
+          'data': {
+            'channel_id': '<channel_id>'
+          },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          }
+        }
+        ```
       - config_changed
+        ```
+        {
+          'event': 'config_changed',
+          'data': {
+            'config': {
+              'EnableCustomEmoji': '<true/false>',
+              'EnableLinkPreviews': '<true/false>'
+            }
+          }
+        }
+        ```
       - delete_team
+        ```
+        {
+          'teamname': '<team_name>',
+          'teamid': '<team_id>',
+          'state': '<success/error>'
+        }
+        ```
       - direct_added
+        ```
+        {
+          'event': '<direct_added>',
+          'seq': '<seq_id>'
+          'data': {
+            'teammate_id': '<teammate_id>'
+          },
+          'broadcast': {
+             'omit_users': null,
+             'user_id': '<user_id>',
+             'channel_id': '<channel.id>',
+             'team_id': '<team_id>'
+          }
+        }
+        ```
       - emoji_added
+        ```
+        {
+          'event': 'emoji_added',
+          'seq': '<seq_id>'
+          'data': {
+            'emoji': {
+              'id': '<id>',
+              'create_at': '<timestamp>',
+              'update_at': '<timestamp>',
+              'delete_at': '<timestamp>',
+              'creator_id': '<creator_id>',
+              'name': 'name'
+            }
+          },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '',
+            'channel_id': '',
+            'team_id': ''
+          }
+        }
+        ```
       - ephemeral_message
       - group_added
       - hello
       - leave_team
+        ```
+        {
+          'event': 'leave_team',
+          'seq': '<seq_id>'
+          'data': {
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          }
+        }
+        ```
       - license_changed
-      - memberrole_updated
+        ```
+        {
+          'type': 'LICENSE_CHANGED',
+          'data': {
+            'license': {
+              'IsLicensed': '<true/false>'
+            }
+          }
+        }
```

Comment:
This one should have a `seq_id` and `broadcast` right?

Reply:
Is that required? https://github.com/mattermost/mattermost-redux/blob/master/src/actions/websocket.test.js#L390